src/fr/mobile/ios/audit-wcag.md

```diff
@@ -7,7 +7,7 @@ abstract: "La vallydette l'outil de test Orange pour mesurer le taux de conformi
 
 L’audit WCAG a pour objectif de calculer les taux de conformité mentionnés sur les déclarations d’accessibilité des applications Orange.
 
-Orange a mis en place une grille de questions nommée la Va11ydette. Cette grille reprend les 50 critères du standard WCAG version 2.1 de niveau A et AA en 52 questions. Chaque question permet de valider ou d’invalider un ou plusieurs critères, un critère peut être traité par une ou plusieurs questions. 
+Orange a mis en place une grille de questions nommée la Va11ydette. Cette grille reprend les 50 critères du standard WCAG version 2.2 de niveau A et AA en 52 questions. Chaque question permet de valider ou d’invalider un ou plusieurs critères, un critère peut être traité par une ou plusieurs questions. 
```

Comment:
'Chaque question permet de valider ou d’invalider un ou plusieurs critères, un critère peut être traité par une ou plusieurs questions.'

Not true.